core/src/components/chip/chip.scss

```diff
@@ -19,13 +20,12 @@
 
   align-items: center;
 
-  height: 32px;
+  min-height: 32px;
```

Comment:
Ok so I figured out why there are screenshot diffs:

The height of the `ion-label` is 17px. This caused the intrinsic height of `ion-chip` to be 17px. Add in the 14px for padding (7px on top 7px on bottom) and 2px for border (1px on top 1px on bottom) and you have 33px.

We previously had `height: 32px` which forced the height of the chip to be `16px` since 16 + 14 + 2 = 32.

We should be able to correct this by reducing the padding from 7px to 6px.